dev/droidmate/projects/core/build.gradle

```diff
@@ -78,15 +78,24 @@ processTestResources.dependsOn("pullApkFixtures", "inlineMonitoredApkFixture")
 //endregion Dependencies
 
 //region Preparation of resources
-
-Project uiaDaemon = findProject(":projects:uiautomator-daemon")
-String uiaDaemonJarName = "${uiaDaemon.name}.jar"
+Project uiaDaemon = findProject(":projects:uiautomator2-daemon")
+String uiaDaemonName = "${uiaDaemon.name}.apk"
+String uiaDaemonTestName = "${uiaDaemon.name}-test.apk"
 String monitorApkName = "monitor.apk"
 
-task pullUiaDaemonJar(type: Copy, dependsOn: ":projects:uiautomator-daemon:build") { Copy copy ->
+task pullUiaDaemon(type: Copy, dependsOn: [":projects:uiautomator2-daemon:assemble", ":projects:uiautomator2-daemon:assembleAndroidTest"]) {
```

Comment:
I simplified this method. Yet you use the old, complex version. Please adapt to use the simplified version, as my code.